customerio.go

```diff
@@ -207,3 +208,48 @@ func (c *CustomerIO) request(method, url string, body interface{}) error {
 
 	return nil
 }
+
+const (
+	// Identifier Types
+	IdentifierTypeID    = "id"
+	IdentifierTypeEmail = "email"
+	IdentifierTypeCioID = "cio_id"
+)
+
+// MergeCustomers sends a request to Customer.io to merge two customer profiles together.
+func (c *CustomerIO) MergeCustomers(primaryIDType, primaryID, secondaryIDType, secondaryID string) error {
```

Comment:
maybe it's time to introduce an `Identifier` struct 
```
type Identifier struct {
   Type IdentifierType
   Value string
}
```

and then this would accept just two params
```
primaryID Identifier, secondaryID Identifier
```